### DISCUSSION TITLE
TotalCMD arguments to specify a configuration file

### DISCUSSION THREAD

**Discussion by BlagoYar** [Flow-Launcher/Flow.Launcher#3146] [Ideas]
Is it possible to add a Total Commander argument field so that you can specify a ini-file
Example:

Field for TotalCMD
_D:\TotalCMD\TotalCMD.exe_

Field for arguments ini file:
_D:\TotalCMD\WinCMD.ini_

**Comment by taooceros** [comment]
I am not sure what do you want to achieve here? If you mean pass the ini to total cmd when used as explorer in flow, I assume you can configure the argument here?
![image](https://github.com/user-attachments/assets/5954805e-2831-4adf-a6c5-67daa5a0fe5b)

**Reply by BlagoYar** [reply]
> I am not sure what do you want to achieve here? If you mean pass the ini to total cmd when used as explorer in flow, I assume you can configure the argument here?

I also thought about it, but for some reason it does not work.
https://youtu.be/cM1704qumhI
